NETProvider/src/FirebirdSql.Data.FirebirdClient/Services/FbServerConfig.cs

```diff
@@ -1,26 +1,29 @@
 /*
- *	Firebird ADO.NET Data provider for .NET and Mono 
- * 
- *	   The contents of this file are subject to the Initial 
- *	   Developer's Public License Version 1.0 (the "License"); 
- *	   you may not use this file except in compliance with the 
- *	   License. You may obtain a copy of the License at 
+ *	Firebird ADO.NET Data provider for .NET and Mono
```

Comment:
Honestly I would like to remove these. But that's probably not possible. I'll probably clean the spaces solution wide with simple PS script.